mg/cmd.go

```diff
@@ -89,6 +93,7 @@ func (w *CmdOutputWriter) dispatch() {
 	}
 }
 
+// Output returns a new CmdOutput object.
```

Comment:
It actually drains the buffer, so I'd saying something like `Output returns the data buffered from previous calls to w.Write() and clears the buffer`